src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -6,46 +6,85 @@
 import net.sf.jabref.model.entry.BibEntry;
 import net.sf.jabref.model.entry.BibtexEntryTypes;
 import org.junit.Assert;
-import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
 import java.nio.charset.StandardCharsets;
 import java.util.List;
 
 public class CopacImporterTest {
 
-    @Before
-    public void setUp() throws Exception {
-        if (Globals.prefs == null) {
-            Globals.prefs = JabRefPreferences.getInstance();
-        }
+    @BeforeClass
+    public static void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
     }
 
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+
     @Test
```

Comment:
I think it is slightly better to understand if you use `@Test(expect=IOException.class)` instead of the `thrown` construction.